src/Hooks/HasPsShippingCostHooks.php

```diff
@@ -101,6 +103,18 @@ private function getDeliveryOptions(Carrier $carrier): DeliveryOptions
             return new DeliveryOptions(json_decode($deliveryOptions, true));
         }
 
+        //Delivery options are not in hidden input, fetch them from the database.
```

Comment:
Zorg dat je formatter on save runt. Plus, als je hier een comment voor nodig hebt is de code _misschien_ niet leesbaar genoeg.

Probeer het eens op een andere manier te organiseren waardoor je niet het gevoel hebt dat je moet uitleggen wat er gebeurt 😄

Reply:
Bijgewerkt